QuickOSM/core/utilities/extent_tool.py

```diff
@@ -0,0 +1,85 @@
+"""A tool that enable to select or show an extent in the canvas."""
+import logging
+
+from qgis.core import (
+    QgsCoordinateReferenceSystem,
+    QgsCoordinateTransform,
+    QgsPointXY,
+    QgsProject,
+    QgsRectangle,
+    QgsWkbTypes,
+)
+from qgis.gui import QgsMapTool, QgsRubberBand
+from qgis.PyQt.QtCore import pyqtSignal
+from qgis.PyQt.QtGui import QColor
+
+__copyright__ = 'Copyright 2021, 3Liz'
+__license__ = 'GPL version 3'
+__email__ = 'info@3liz.org'
+
+LOGGER = logging.getLogger('QuickOSM')
+
+
+class ShowExtent(QgsMapTool):
+    """Show an extent in the canvas"""
+
+    ShowEnded = pyqtSignal()
+
+    def __init__(self, canvas):
+        """Constructor"""
+        QgsMapTool.__init__(self, canvas)
+
+        self.canvas = canvas
+        self.rubberBand = QgsRubberBand(self.canvas, QgsWkbTypes.PolygonGeometry)
```

Comment:
Minor, but Python convention is to use snake case for variable. `rubber_band` https://en.wikipedia.org/wiki/Snake_case